README.adoc

```diff
@@ -45,7 +45,7 @@ docker volume create jenkins-evergreen-data && \
 docker run --name evergreen \
     --restart=always \
     -ti \
-    -p 8080:8080 \
+    -p 8080:80 \
```

Comment:
Interesting issue I just hit because of this: this docs got updated immediately, when for some reason, the public Docker image seems to be still not updated for some reason. So Running the docs as is was binding 8080 to 80, but my local image does not have `nginx` yet. (yes, I did a `docker pull` in case)

Reply:
FTR, the updated image has been updated now.